scripts/cluster/setup_master_node.sh

```diff
@@ -48,7 +48,7 @@ export PATH=$PATH:$ROOT/istio-1.12.5/bin
 sudo sh -c  "echo 'export PATH=\$PATH:$ROOT/istio-1.12.5/bin' >> /etc/profile"
 istioctl install -y -f $ROOT/configs/istio/istio-minimal-operator.yaml
 
-KNATIVE_VERSION="knative-v1.4.0"
+KNATIVE_VERSION="knative-v1.9.0"
```

Comment:
Did you modify the queue-proxy of Knative v1.9 prior to building it? If not, Firecracker mode won't work.

You need to sync the fork of Knative (https://github.com/ease-lab/serving) and checkout to v1.9 and apply the following changes:
- https://github.com/ease-lab/serving/commit/8ab7e6985be0bcf8263c347a6f02148e492f63be
- https://github.com/ease-lab/serving/commit/4aac0b2b5b1bb901d69297083d2e6421ad659e02

Then you can build Knative and change the YAML files in `configs/knative_yamls`.

Reply:
Yes, I did modify the queue-proxy of Knative v1.9, I've updated the Knative upgrade [Wiki ](https://github.com/vhive-serverless/vHive/wiki/TMP:-How-to-Update-Knative-for-vHive) as well with the appropriate instructions. I've generate the yamls with the changes on this [PR](https://github.com/ease-lab/serving/pull/10).

Reply:
Looks good. Have you updated YAMLs in this PR to include changes in the queue-proxy? Once that is done, please test the whole setup with instructions from the quick start guide, insect statuses of pods in K8s, check there aren't errors (post the output here), and run some workload. If that passes, we are good to merge the PR.

Reply:
Yes, I did update the YAMLs in this PR. I've tested it and everything works as expected.